dist/css_tagger.min.js

```diff
@@ -1,3 +1 @@
-// Minified on http://closure-compiler.appspot.com/ using Advanced settings
```

Comment:
I didn't use that minifier. I used [uglifyjs](https://www.npmjs.com/package/uglify-js) with the `--mangle` option.

Is there some reason that one is preferred?